README.md

```diff
@@ -63,13 +63,20 @@ default_cluster: "YOUR_CLUSTER"
 ```
 
 ## Scheduling a job
-You are now ready to schedule jobs!
+You are now ready to schedule jobs! To run the following examples, make sure to install all dependencies:
+
+```bash
+pip install -e ".[extra]"
+```
+
 Let us have a look at `launch_hellocluster.py`, in particular, you can call the following to schedule a job:
 
 ```python
-cluster, partition = default_cluster_and_partition()
+user_path = Path('slurmpilot/config')
```

Comment:
By default, user_path is set to `~/slurmpilot/config`, is there a benefit to not use this by default? (Also I would be keen to avoid paths that are dependent from the cwd)